go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/prometheus/common v0.37.0
 	github.com/prometheus/exporter-toolkit v0.7.1
 	github.com/robfig/cron/v3 v3.0.1
-	github.com/spf13/pflag v1.0.5
+	github.com/spf13/cobra v1.5.0
```

Comment:
sorry, this has been sitting for a while without review. Can you update to cobra 1.6.1? 
There have been some changes around the --help and --version flags which might be interesting in v1.6.0 https://github.com/spf13/cobra/releases/tag/v1.6.0